File: 진우/10weeks/2751. 수 정렬하기 2/수 정렬하기 2.py
import sys

# 빠른 입력 (sys.stdin.readline 사용)
N = int(sys.stdin.readline())

arr = []  # 빈 리스트 생성

# N개의 정수를 입력받아 리스트에 추가
for _ in range(N):
    x = int(sys.stdin.readline())  # 한 줄씩 입력받기
    arr.append(x)

# 리스트를 오름차순으로 정렬
arr.sort()

# join을 이용해 한 번에 출력 (빠른 출력)
print("\n".join(map(str, arr)))

File: 진우/10weeks/2751. 수 정렬하기 2/수 정렬하기 2.cc
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int N;
    cin >> N;

    vector<int> arr(N); // N개의 원소를 저장할 벡터 생성

    // 값 입력 (push_back이 아니라 인덱스 접근으로 입력)
    for (int i = 0; i < N; i++) {
        cin >> arr[i];
    }

    // 오름차순 정렬
    sort(arr.begin(), arr.end());

    // 정렬된 결과 출력
    for (int i = 0; i < N; i++) {
        cout << arr[i] << "\n";
    }

    return 0;
}
